tools/createMissingUsers.js

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This file was copied from the original `create_missing_users.js` so it should do the same thing.
The difference is that its using async/await, fetch, and mongoose